### PR TITLE
DOC: Edited doc string of pandas/core/frame.duplicated(). Redefined take...

### PR DESCRIPTION
**Pull request by robertdavidwest**
..._last variable in doc string. Original definition only made sense for drop_duplicates().

### CODE REVIEW

File: pandas/core/frame.py [hunk 1]
```diff
@@ -2726,7 +2726,8 @@ def duplicated(self, subset=None, take_last=False):
             Only consider certain columns for identifying duplicates, by
             default use all of the columns
         take_last : boolean, default False
-            Take the last observed row in a row. Defaults to the first row
+             For a set of distinct duplicate rows, flag all but the last row as duplicated. 
+             Default is for all but the first row to be flagged
```
Comment (jorisvandenbossche) [inline pandas/core/frame.py]: Very small detail: I think your new line is one space off in indentation

Comment (robertdavidwest) [inline pandas/core/frame.py]: Thank you. Amended.

**Comment by shoyer** [comment]
xref: #6511, #8505

looks good to me

@robertdavidwest could you please squash these commits together?

**Comment by jorisvandenbossche** [comment]
Follow-up: #9206

**Pull request closed without merging**